metagraph/src/cli/config/config.cpp

```diff
@@ -422,7 +422,7 @@ Config::Config(int argc, char *argv[]) {
         parallel_nodes = get_num_threads();
 
     if (identity == TRANSFORM && to_fasta)
-        identity = ASSEMBLE;
+        identity = CLEAN;
```

Comment:
Transforming to fasta may be done for either assembly or cleaning, so the user's intention is not clear here. Can we just get rid of `transform --to-fasta` support and tell the user to use either `clean` or `assemble` instead?

Reply:
personally, I think of "transform" as meaning "graph -> graph" and "assemble" as "graph -> sequence set", but we can probably save this discussion for a different pull request